README.md

```diff
@@ -1,2 +1,78 @@
-# onboarding
-Engineering onboarding
+Onboarding
+==========
+
+This repo defines the Hypothesis new engineer onboarding process:
+
+* [Issue templates](.github/ISSUE_TEMPLATE) in this repo help to create the
+  onboarding issues for each new engineer
+* [Project boards](https://github.com/hypothesis/onboarding/projects) in this
+  repo keep track of each new engineer's onboarding progress
+* [The docs folder](docs) in this repo contains the documentation and guides
+  that a new engineer will need: our architecture, core practices, etc
+* To report a problem or suggest an improvement to the onboarding process
+  [open an issue](https://github.com/hypothesis/onboarding/issues/new) or
+  [create a pull request](https://github.com/hypothesis/onboarding/compare)
+
+Creating an onboarding project for a new engineer
+-------------------------------------------------
+
+Before a new engineer's first day we need to create a
+[GitHub project](https://docs.github.com/en/issues/planning-and-tracking-with-projects)
+for their onboarding. To create an onboarding project:
+
+1. [Create a new project](https://github.com/orgs/hypothesis/projects) in the Hypothesis organization
+   (there are no repo projects: all projects in GitHub are organization-level).
+   Choose the simpler **Board** layout, the more powerful **Table** layout is
+   too much for an onboarding project.
+
+2. Give the new project a title/name like **@seanh Onboarding**.
+
+3. Add the new project to this onboarding repo for quick access:
+   go to [the onboarding repo's **Projects** tab](https://github.com/hypothesis/onboarding/projects)
+   and use the **Add project** button.
+
+4. Create issues in the onboarding repo and add them to the project:
+
+   1. Go to the onboarding repo's [new issue page](https://github.com/hypothesis/onboarding/issues/new/choose)
+      and create an issue from one of the templates or use the **Open a blank issue** link:
+
+      ![Choosing an issue template](docs/images/choosing_an_issue_template.png)
+
+   2. Use the **Assignees** section in the issue's sidebar to assign the issue
+      to the new engineer and the **Projects** section to add the issue to the
+      project:
```

Comment:
```suggestion
      onboarding project you created for the new engineer:
```

I suggest this because the word "project" alone might be confused with this repo or other projects.